articles/azure-app-configuration/monitor-app-configuration-reference.md

```diff
@@ -26,6 +26,7 @@ Resource Provider and Type: [App Configuration Platform Metrics](../azure-monito
 | Daily Storage Usage | Percent |	Represents the amount of storage in use as a percentage of the maximum allowance. This metric is updated at least once daily. |
 | Request Quota Usage | Percent |	Represents the current total request usage in percentage. |
 | Replication Latency | Milliseconds |	Represents the average time it takes for a replica to be consistent with current state. |
+| Snapshot Storage Size | Count | Represents the total storage usage of key value snapshot(s) in bytes. |
```

Comment:
> key value snapshot(s)

We normally call it "configuration snapshot(s)". Is the current name from somewhere?

In the "Daily Storage Usage", we explained how frequently the metric is updated, should we do the same here?

Customer may feel confused with this metric vs. the "Daily Storage Usage". Should we explain the difference?

Reply:
For the name, I referred to [this ](https://microsoft-my.sharepoint.com/:w:/g/personal/xuxu_microsoft_com1/EWqgnHmuJiRDgiahR4H_xggBipg9a5if2kzac9iT8cS4GA?e=2uGInS)doc. The metric is updated every 24 hours (similar to Daily Storage Usage), updated the definition.